clap_derive/src/utils/doc_comments.rs

```diff
@@ -103,5 +103,5 @@ fn is_blank(s: &str) -> bool {
 }
 
 fn merge_lines(lines: &[&str]) -> String {
-    lines.iter().map(|s| s.trim()).collect::<Vec<_>>().join(" ")
+    lines.iter().map(|s| s.trim().replace("\\n", "\n")).collect::<Vec<_>>().join(" ")
```

Comment:
> Note: this PR doesn't allow to escape slash, so \\n will be rendered as \{newline}

I'm always a bit cautious of doing our own escaping.  We should be matching Rust's rules for escape sequences if we do this.

Reply:
Fixed in next commit: now `\\\\` will be rendered as `\\` and `\\n` will be rendered as `\n`.

Although, I still don't recommend to merge this PR